src/v/kafka/server/handlers/fetch.cc

```diff
@@ -149,6 +175,24 @@ static ss::future<read_result> read_from_partition(
 
     reader_config.strict_max_bytes = config.strict_max_bytes;
     auto rdr = co_await part.make_reader(reader_config);
+
+    try {
+        auto script_with_env = co_await get_wasm_script(
+          v8_scripts_dispatcher, ntp);
+        if (script_with_env.has_value()) {
+            rdr = co_await std::move(rdr).consume(
+              v8_engine::v8_batch_consumer<v8_engine::executor_wrapper>(
+                std::move(script_with_env.value())),
+              deadline ? *deadline : model::no_timeout);
+        }
+    } catch (const std::runtime_error& ex) {
+        vlog(klog.error, "Error in inline transformation {}", ex.what());
+        co_return error_code::inline_wasm_error;
```

Comment:
Can I create new error_code

Reply:
No we can't  -- I added a comment where this error code was added.